provider.tf

```
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5"
+    }
+  }
+}
+
+provider "aws" {
+  profile = "terraform-test"
+  region  = local.region
+}
```

In time we'll setup a role to assume here in our provider config specific to terraform in each account. We'll then have permissions to assume it individually and can also grant any CI permission to assume the same role. That way this provider block will work for all of us who have permission, regardless of our individual profile names. 

For now though, I think let's introduce a common profile name we can all use that represents the account and the role. I believe this is `Administrator-LongSession` under the `SRE-ENPLAT-GX-DEV` account, so maybe if we each configure consistently named local profiles in the format `<account-name>-<role-name>`? Using the full values will be a bit unwieldy when setting your profile, so I think we simplify further since we only have two accounts we're using this way at the moment - `enplat-dev-admin` and `enplat-prod-admin`. Open to your suggestions here too -m we can always refine later also, or it will naturally become better late through a named role we all assume anyway.

```suggestion
provider "aws" {
  profile = "enplat-dev-admin"
  region  = local.region
}
```

---

This looks like a clean approach to access the profile for all. New profile is configured for the code, however access for the account COP-AWS3-IE-SBX (654654301190) with Administrator-LongSession is required to use the associated profile.